src/apps/fl-content/style.scss

```diff
@@ -0,0 +1,29 @@
+.fl-asst {
```

Comment:
Should these styles live with the detail view instead of the app?

Reply:
I moved to pages/post/style.scss